### PR TITLE
Build Errors

### PR DESCRIPTION
**Pull request by LucaHaverty**
### Description
Fixed build errors caused by preferences being of type 'object' and what I assume was a merge conflict to do with sliders

### CODE REVIEW

File: fission/src/ui/panels/configuring/scoring/ZoneConfigPanel.tsx [hunk 1]
```diff
@@ -40,7 +39,7 @@ const ZoneConfigPanel: React.FC<PanelPropsImpl> = ({ panelId, openLocation, side
             <NumberInput label="Points" placeholder="Zone points" defaultValue={1} onInput={v => setPoints(v || 1)} />
             <Checkbox label="Destroy Gamepiece" defaultState={false} onClick={setDestroy} />
             <Checkbox label="Persistent Points" defaultState={false} onClick={setPersistent} />
-            <Slider
+            {/* <Slider
```
Comment (Dhruv-0-Arora) [inline fission/src/ui/panels/configuring/scoring/ZoneConfigPanel.tsx:42]: Note: Sliders being commented will be overridden by PR #1021

File: fission/src/ui/panels/configuring/scoring/ZoneConfigPanel.tsx [hunk 2]
```diff
@@ -22,11 +21,11 @@ const ZoneConfigPanel: React.FC<PanelPropsImpl> = ({ panelId, openLocation, side
     // then set all default values to the state of the zone
     const [, setName] = useState<string>("")
     const [alliance, setAlliance] = useState<"red" | "blue">("blue")
-    const [, setParent] = useState<string>("")
+    const [,] = useState<string>("")
```
Comment (Dhruv-0-Arora) [inline fission/src/ui/panels/configuring/scoring/ZoneConfigPanel.tsx:24]: Note: Also going to be overriden by Zone Config PR

**Review by Dhruv-0-Arora — APPROVED** [review]
It runs 👍 ; Magnificent

**Review by a-crowell — APPROVED** [review]
It builds!

**Pull request merged**